clang/test/Sema/enum.c

```diff
@@ -169,13 +169,19 @@ enum class GH42372_2 {
   One
 };
 
+enum IncOverflow {
+  V2 = __INT_MAX__,
+  V3 // c99-warning {{incremented enumerator value that is exceeding range of 'int' is a C23 extension}}
+#if __STDC_VERSION__ >= 202311L
```

Comment:
You can give the C23 RUN line its own `-verify` prefix rather than using `#if`

Reply:
I missed default ignore for the warning. Now it is tested by another test and doesn't appear here.